HintList.py

```diff
@@ -620,6 +622,7 @@ def tokens_required_by_settings(world: World) -> int:
     'Graveyard Royal Family Tomb Contents':                        ("inside the #Royal Family Tomb#, you will find...^", None, 'dual'),
     'DMC Child Upper Checks':                                      ("in the #crater, a spider in a crate and a single scrub# guard...^", None, 'dual'),
     'Haunted Wasteland Checks':                                    ("deep in the #wasteland a spider and a chest# hold...^", None, 'dual'),
+    'King Zora Items':                                             ("#unfreezing King Zora and giving him Prescription# rewards both...^", None, 'dual'),
```

Comment:
```suggestion
    'King Zora Items':                                             ("#unfreezing King Zora and giving him the Prescription# rewards...^", None, 'dual'),
```
Removed the “both” which has been removed from other dual hints in the past. Added “the” which sounds better imo.